inst/rmarkdown/templates/ieee_article/resources/template.tex

```diff
@@ -334,14 +334,14 @@
 \usepackage{natbib}
 \bibliographystyle{$if(biblio-style)$$biblio-style$$else$plainnat$endif$}
 $endif$
```

Comment:
@swhaat @DunLug as it seems `biblatex` should be use by default, should this be removed ? 
It seems to me it can cause issue with the new default if someone provide `natbib` option.

Reply:
It seems that it could be conflict in that case. If we want to allow the natbib option, we could replace these lines with an else statement :
$if(natbib)$
\usepackage{natbib}
\bibliographystyle{$if(biblio-style)$$biblio-style$$else$plainnat$endif$}
$else$
\usepackage[backend=bibtex,citestyle=ieee]{biblatex}

$for(bibliography)$
\addbibressource{$bibliography$}

In that case, maybe another conflict could occur on lines 634-635. As the if statement should only occur with biblatex package, we could replace these with : 
$if(natbib)$
$else$
$if(bibliography)$
\printbibliography[heading=none]
$endif$
$endif$
$endfor$
$endif$
$endif$

I wonder if we could harmonize the natbib and biblatex behaviors by removing the biblio-style option in natbib